state/carmen.go

```diff
@@ -160,6 +164,21 @@ func (s *carmenStateDB) SetState(addr common.Address, key common.Hash, value com
 	s.txCtx.SetState(carmen.Address(addr), carmen.Key(key), carmen.Value(value))
 }
 
+func (s *carmenStateDB) GetStorageRoot(addr common.Address) common.Hash {
+	panic("GetStorageRoot not implemented")
+	return common.Hash{}
+}
+
+func (s *carmenStateDB) GetTransientState(addr common.Address, key common.Hash) common.Hash {
+	panic("GetTransientState not implemented")
+	return common.Hash{}
+}
+
+func (s *carmenStateDB) SetTransientState(addr common.Address, key common.Hash, value common.Hash) {
+	panic("SetTransientState not implemented")
+	return
+}
```

Comment:
This is already implemented [here](https://github.com/Fantom-foundation/Carmen/blob/ad1657431037430a60f7456088947e5bfd16972a/go/carmen/transaction.go#L115)

Reply:
Yes, however, I have to refer to the older version of Carmen which matches the one in Sonic to resolve dependency issue. The new facades of carmen are not available.